docs/source/sctool/backup.rst

```diff
@@ -35,175 +35,8 @@ You cannot initiate a backup without a cluster. Make sure you add a cluster (:re
 backup
 ======
 
```

Comment:
@mmatczuk The command ``sctool_backup_validate_update.yaml`` is not in the docs (rst) file. Should I create a new heading and include it?